message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -241,6 +249,36 @@
         <param index="6">Reserved, send 0</param>
         <param index="7">Reserved, send 0</param>
       </entry>
+      <entry value="43000" name="MAV_CMD_GUIDED_CHANGE_SPEED">
```

Comment:
A comment here as to when you'd actually use these would be handy.  Or improve the descriptions as "Change target altitude" is the same as (several?) other messages.